chromeos.kernelci.org

```diff
@@ -0,0 +1,224 @@
+}
+
+cmd_buildroot() {
+    echo "Updating buildroot"
+    ./pending.py buildroot --settings=${CHROMEOS_SETTINGS} --main=chromeos.kernelci.org --push
+}
+
+cmd_test_definitions() {
+    echo "Updating test-definitions"
+    ./pending.py test-definitions --settings=${CHROMEOS_SETTINGS} --main=chromeos.kernelci.org --push
+}
+
+cmd_backend() {
+    opt="$1"
+
+    echo "Checking for missing indexes by finding COLLSCAN events"
+    echo "db.system.profile.aggregate([{\$match: {ns: {\$in: ['chromeos-kernel-ci.test_case', 'chromeos-kernel-ci.test_group', 'chromeos-kernel-ci.test_regression', 'chromeos-kernel-ci.build']}, planSummary: 'COLLSCAN'}}, {\$group: {_id: {ns: '\$ns', planSummary: '\$planSummary'}}}])" | mongo chromeos-kernel-ci
+
+    if [ "$opt" = "indexes" ]; then
+        echo "Dropping Mongo DB indexes"
+        echo "db.test_case.dropIndexes(); db.test_group.dropIndexes(); db.test_regression.dropIndexes(); db.build.dropIndexes();" | mongo chromeos-kernel-ci
+    fi
+
+    echo "Updating kernelci-backend"
+    ./pending.py kernelci-backend --settings=${CHROMEOS_SETTINGS} --main=chromeos.kernelci.org --push
+    # ToDo: check if there were any changes to skip if there weren't any
+    ./ansible kernelci-backend api.chromeos.kernelci.org
+
+    echo "Resetting and enabling the Mongo DB profiler"
+    echo "db.setProfilingLevel(0); db.system.profile.drop(); db.setProfilingLevel(2);" | mongo chromeos-kernel-ci
+}
+
+cmd_frontend() {
+    echo "Updating kernelci-frontend"
+    ./pending.py kernelci-frontend --settings=${CHROMEOS_SETTINGS} --main=chromeos.kernelci.org --push
+    # ToDo: check if there were any changes to skip if there weren't any
+    ./ansible kernelci-frontend chromeos.kernelci.org
+}
+
+cmd_website() {
+    echo "Updating static website"
+    ./pending.py kernelci-project --settings=${CHROMEOS_SETTINGS} --main=chromeos.kernelci.org --push
+    topd=$PWD
+    cd checkout/kernelci-project/kernelci.org
+    git-lfs checkout
+    git submodule update --init --recursive
+    for d in external/*; do
+        echo "Checking out staging branch for $d"
+        cd "$d"
+        git remote update origin
+        git checkout origin/staging.kernelci.org
+        cd -
+    done
+    ./docker-hugo -D -b https://chromeos.kernelci.org:8080
+    cd "$topd"
+}
+
+cmd_kernel() {
+    tree="${1:-next}"
+
+    echo "Pushing kernel test branch for ${tree}"
+
+    case $tree in
+        next)
+            url=https://git.kernel.org/pub/scm/linux/kernel/git/next/linux-next.git
+            branch=master
+            ;;
+        mainline)
+            url=https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git
+            branch=master
+            ;;
+        stable)
+            url=https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux-stable.git
+            branch=linux-5.4.y
+            ;;
+        *)
+            echo "Unknown tree"
+            exit 1
+            ;;
+    esac
```

Comment:
As discussed, please drop this as we're starting with only one kernel branch (stable 5.12) on chromeos.kernelci.org.